tests/e2e/specs/modules/tagmanager/setup.test.js

```diff
@@ -59,6 +59,11 @@ describe( 'Tag Manager module setup', () => {
 				request.respond( {
 					status: 200,
 				} );
+			} else if ( request.url().match( 'google-site-kit/v1/core/site/data/notifications' ) ) {
+				request.respond( {
+					status: 200,
+					body: JSON.stringify( [] ),
+				} );
```

Comment:
This was the request that was often causing the test to fail as an unexpected API failure (or at least one of them 😄 ).